### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by kradl**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+language: java
+jdk:
+  - oraclejdk8
```
Comment (kernelgonnapanic) [inline .travis.yml:3]: yyyyy?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+node_js: '6'
+env:
+  - NODE_VERSION=6.9.1
+script:
+  - nvm use $NODE_VERSION
+  - npm run unit
+before_install:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```
Comment (kernelgonnapanic) [inline .travis.yml:12]: Why do we need this?

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "files.associations": {
+        "*.vue": "vue"
+    }
+}
```
Comment (kernelgonnapanic) [inline .vscode/settings.json]: An empty line needed

File: src/components/ProductsList.vue [hunk 4]
```diff
@@ -21,63 +21,63 @@
         description: 'Sweet tart sweet toffee brownie ice cream toffee cookie.' +
           'Toffee halvah chocolate bar donut. Candy cake brownie jelly bonbon biscuit.',
         price: 2000,
-        photo: 'http://lorempixel.com/300/200/cats/1',
+        photo: 'http://placehold.it/300x200',
```
Comment (kernelgonnapanic) [inline src/components/ProductsList.vue:24]: ?

File: .travis.yml (hunk 1 above)
Comment (kradl) [reply .travis.yml:3]: selenium 3.X requires java8, which is not available to chose in "node_js" language setting in travis

File: .travis.yml (hunk 2 above)
Comment (kradl) [reply .travis.yml:12]: for the future to run not the headless one browsers like chrome

File: src/components/ProductsList.vue (hunk 4 above)
Comment (kradl) [reply src/components/ProductsList.vue:24]: lorempixel pictures loaded to slow, causing to e2e tests timeout

**Comment by Qwal** [comment]
squash before merge pls

**Pull request merged**